spec/header-files.md

```diff
@@ -21,28 +21,35 @@ A header file contains:
 
 Each line must be under 255 characters, and fields are separated by spaces or tabs (except where otherwise noted).
 
+Detailed documentation on header files can be found at: [https://physionet.org/physiotools/wag/header-5.htm](https://physionet.org/physiotools/wag/header-5.htm)
+
 ---
 
 ## Record Line
 
 The first non-comment line is the **record line**, which provides metadata about the overall record. It includes:
 
-| Field | Description |
-|:------|:------------|
-| Record name | Identifier for the record (letters, digits, underscores only). |
-| Number of segments (optional) | If present, appended as `/n`. Indicates a multi-segment record. |
-| Number of signals | Number of signals described in the header. |
-| Sampling frequency (optional) | Samples per second per signal. Defaults to 250 if omitted. |
-| Counter frequency (optional) | Secondary clock frequency, separated from sampling frequency by a `/`. |
-| Base counter value (optional) | Offset value for counter, enclosed in parentheses. |
-| Number of samples (optional) | Total samples per signal. |
-| Base time (optional) | Start time of the recording (`HH:MM:SS`). |
-| Base date (optional) | Start date (`DD/MM/YYYY`). |
+| Field | Description                                                                |
+|:------|:---------------------------------------------------------------------------|
+| Record name | Identifier for the record (letters, digits, underscores only).             |
+| Number of segments (optional) | If present, appended as `/n`. Indicates a multi-segment record.            |
+| Number of signals | Number of signals described in the header.                                 |
+| Sampling frequency (optional) | Samples per second per signal. Defaults to 250 if omitted.                 |
+| Counter frequency (optional) | Secondary clock frequency, separated from sampling frequency by a `/`.     |
+| Base counter value (optional) | Offset value for counter, enclosed in parentheses.                         |
+| Number of samples (optional) | Total samples per signal when `samps_per_frame`=1; total frames otherwise. |
+| Base time (optional) | Start time of the recording (`HH:MM:SS`).                                  |
+| Base date (optional) | Start date (`DD/MM/YYYY`).                                                 |
 
 **Example:**
 ```text
-100 2 360 650000 12:00:00 01/01/2000
+12345 3 62.5 625 30/01/1989
```

Comment:
In the new example, we have lost base time but for no particular reason I can see.  Instead of overwriting the existing sample, I wonder whether it would be better to list multiple examples? 

My preference is to use real-life examples if we can, so that we could link to the real file allowing people to do a deeper exploration.